RecoTauTag/HLTProducers/interface/L1THLTJetsMatching.h

```diff
@@ -0,0 +1,36 @@
+#ifndef L1THLTJETSMATCHING_H
+#define L1THLTJETSMATCHING_H
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/L1Trigger/interface/Jet.h"
+#include "DataFormats/JetReco/interface/CaloJetCollection.h"
+#include "DataFormats/HLTReco/interface/TriggerFilterObjectWithRefs.h"
+
+
+#include <map>
+#include <vector>
+
+class L1THLTJetsMatching: public edm::EDProducer {
+ public:
+  explicit L1THLTJetsMatching(const edm::ParameterSet&);
+  ~L1THLTJetsMatching();
+  virtual void produce(edm::Event&, const edm::EventSetup&) override;
+
+ private:
+  l1t::JetVectorRef tauCandRefVec;
+  l1t::JetVectorRef jetCandRefVec;
+  l1t::JetVectorRef objL1CandRefVec;
+  l1t::JetRef tauCandRef;
+
```

Comment:
Please do not remove any modules from 8.0.x now.

(it's a closed release, and leaving the old modules around won't break
anything; no objections to removing them from 8.1x, though, if they are not
used)

.A